engineering/passports/id/DESIGN_DOC.md

```diff
@@ -59,20 +65,28 @@ Each user can have an arbitrary number of passports associated with them but onl
 
 A basic interface reminiscent of other current sign in platforms will ask the user if they want to approve the connection with the target app and list the scopes the app is requesting after they verify ownership of their passport. Passport ownership is verified by entering the passport number then scanning the passport with a phone to contact the server and allow the sign in.
 
+The user management system will allow a user to enroll/unenroll from 2FA and manage aspects of their profile as well as adding their own clients to work with ID. The admin side will allow arbitrary user operations (enable/disable, see 2FA, etc).
```

Comment:
Users adding their own clients should have restrictions to prevent access to sensitive information that should only be kept within Purdue Hackers applications, but is otherwise a good idea